AspNetCore.SassCompiler.Tests/SassCompilerTests.cs

```diff
@@ -32,6 +32,32 @@ public async Task CompileAsync_WithoutStreams_Success()
         }
     }
 
+    [Fact]
```

Comment:
Can you set a timeout on this test, so that the tests will fail if it takes too long (1 second should probably be enough)